apps/voting/contracts/Voting.sol

```diff
@@ -306,45 +415,38 @@ contract Voting is IForwarder, AragonApp {
     * @dev Internal function to cast a vote. It assumes the queried vote exists.
     */
     function _vote(uint256 _voteId, bool _supports, address _voter, bool _executesIfDecided) internal {
```

Comment:
Could we change most of the internal and private functions to use `Vote storage _vote` instead?

Reply:
I did with most of them, but I decided not to change those that are actually relying on the id. For example, if we change this function, we will need to pass both the id and the storage pointer, which could lead to undesired behaviors if someone uses it wrongly. An alternative could be to turn these cases into `private` functions to ensure we're the only ones calling those. WDYT?

Reply:
Ah yes, that makes sense. I see most of these have to emit events, which require the ID 👍.